modules/node_modules/@ncigdc/containers/FileTable.js

```diff
@@ -139,6 +139,7 @@ export const FileTableQuery = {
                 total
```

Comment:
I would only ask for 1 case since you only need the actual data from the first hit